rest/cache.py

```diff
@@ -69,7 +70,7 @@ def get_int_setting(param, default):
     enabled = False
 
 
-def cache():
+def cache(userSpecific=False):
```

Comment:
> Userspecific only when privateData is on?

This doesn't solve the mentioned problem neither.

For me, the right solution is to manage the cache at pyslurm calls level, not at API route level. This is what I suggested last friday.